components/sidebar/sidebar.tsx

```diff
@@ -222,7 +224,7 @@ export default class Sidebar extends React.PureComponent<Props, State> {
         const ariaLabel = Utils.localizeMessage('accessibility.sections.lhsNavigator', 'channel navigator region');
 
         return (
-            <div
+            <Resizable
```

Comment:
Would you mind renaming the `Resizable` components to `ResizableLhs` and `ResizableRhs` so that we can tell them apart?